core/src/main/java/com/datastrato/gravitino/GravitinoEnv.java

```diff
@@ -74,7 +75,9 @@ public void initialize(Config config) {
     // Initialize EntityStore
     this.entityStore = EntityStoreFactory.createEntityStore(config);
     entityStore.initialize(config);
-    entityStore.setSerDe(entitySerDe);
+    if (!(entityStore instanceof RelationEntityStore)) {
```

Comment:
I prefer setting the `serde `in the method `initialize`.

Reply:
Let me try moving this into the `initialize` method. For relational entity store, this may not be necessary.

Reply:
Yes, I know. Just want to reduce the similar code.
```
if (!(entityStore instanceof RelationEntityStore)) {
}
```

Reply:
I found that there are many places where this method is called. I think we can make this change separately in a new PR. What do you think?

Reply:
I don't want to remove `instanceof` in the other places. I just want to remove it in this place. I just feel it's not necessary here.

Reply:
done